loadtest/control/simulcontroller/actions.go

```diff
@@ -661,6 +672,10 @@ func (c *SimulController) createPost(u user.User) control.UserActionResponse {
 		return control.UserActionResponse{Err: control.NewUserError(err)}
 	}
 
+	if shouldLongThread {
+		st.setLongRunningThread(postId, channel.Id, channel.TeamId)
+	}
```

Comment:
Its a bit tricky because I don't have the final `postID` returned from `CreatePost`. I'll need the post id in `createMessage` to set it as a long running thread.

Reply:
Fair enough, let's leave it.